examples/client/client.c

```diff
@@ -2737,8 +2739,8 @@ THREAD_RETURN WOLFSSL_THREAD client_test(void* args)
     if (doMcast) {
 #ifdef WOLFSSL_MULTICAST
         byte pms[512]; /* pre master secret */
-        byte cr[32];   /* client random */
-        byte sr[32];   /* server random */
+        byte cr[MSG32];   /* client random */
+        byte sr[MSG32];   /* server random */
```

Comment:
This is 32 by TLS specification and is not related to the message length MSG32. I'd suggest reverting these two lines.

Reply:
It's part of our coding standard to try and avoid magic numbers whenever possible. I just setup a new macro MSG32 to replace occurrences of the magic number "32" in array declarations. There are other magic numbers in the example but this isn't an exhaustive cleanup so I only updated ones consistent with the change.

Reply:
@kaleb-himes : I agree with @Lekensteyn this is mixing two different things. If MSG32 gets changed then it could break the expected TLS random size. Also the macro name should not have the number in it... It should be something like `MSGSZ` and `GETMSGSZ`. These should be using something like our internal.h `RAN_LEN`. Please issue another PR to correct these.

Reply:
@dgarske: https://github.com/wolfSSL/wolfssl/pull/3081/files